pkg/protocol/selfdescribing/envelopeBuilder.go

```diff
@@ -21,8 +24,24 @@ func buildEnvelopesFromRequest(c *gin.Context, conf *config.Config, m *meta.Coll
 	reqBody, err := io.ReadAll(c.Request.Body)
 	if err != nil {
 		log.Error().Err(err).Msg("🔴 could not read request body")
+		util.Pprint(c.Request.Body)
```

Comment:
let's pop this out of here as well, `util.Pprint` is mostly a helper utility, `log.Error` should cover what we need.